examples/client-logger/app-insights-logger/package.json

```diff
@@ -53,6 +56,7 @@
 		"react-dom": "^17.0.1"
 	},
 	"devDependencies": {
```

Comment:
You'll need to add a dev dependency on the "previous" version of this package. Take a look at one of our other packages using the type tests for an example.